fcrepo-http-api/src/main/java/org/fcrepo/http/api/FedoraBaseResource.java

```diff
@@ -101,7 +90,7 @@ protected FedoraResource getFedoraResource(final Transaction transaction, final
      * @return Returns true if an object with the provided id exists
      */
     protected boolean doesResourceExist(final Transaction transaction, final FedoraId fedoraId) {
-        return resourceFactory.doesResourceExist(transaction, fedoraId);
+        return resourceFactory.doesResourceExist(TransactionUtils.openTxId(transaction), fedoraId);
```

Comment:
Would it make sense to also add back in the `doesResourceExist(Transaction tx, ...)` signature, which internally calls `doesResourceExist(String txId, ...)`?